.circleci/config.yml

```diff
@@ -284,7 +284,7 @@ commands:
         type: string
       no_output_timeout:
         type: string
-        default: 30m
```

Comment:
Have this been discussed with devops?

Reply:
This job is not merging with master anytime soon :-) 

This change is necessary for the branch to pass the tests on circleci. 
New cryptographic algorithms introduced in this PR are significantly more expensive, so many tests require more time. 

This parameter change will be part of a bigger solution that we are still working on to resolve the testing time.

Reply:
sounds good, lets discuss on discord. It is ok while not merged to master.

Reply:
Reverting the changes for now. If something comes up, we will consult you. thanks